tutorials/lsrro.ipynb

```diff
@@ -0,0 +1,5428 @@
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'Membrane CAPEX': [0.07201792002054254],\n",
+       " 'Pump CAPEX': [0.49818640011410525],\n",
+       " 'ERD CAPEX': [0.032699960698051235],\n",
+       " 'Indirect CAPEX': [0.6029042808326991],\n",
+       " 'Membrane OPEX': [0.1512376320431393],\n",
+       " 'Electricity': [0.7309236776200411],\n",
+       " 'Other OPEX': [0.3185318164872939]}"
+      ]
+     },
+     "execution_count": 10,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "lcow_breakdown_results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 5: Optimize the LSRRO flowsheet."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lsrro.optimize_set_up(m,\n",
+    "    water_recovery=0.5,\n",
+    "    A_case=ACase.optimize, # water permeability coefficient\n",
+    "    B_case=BCase.optimize, # salt permeability coefficient\n",
+    "    AB_tradeoff=ABTradeoff.equality_constraint, # water and salt permeability equality constraints\n",
+    "    permeate_quality_limit=1000e-6,\n",
+    "    )\n",
+    "optimization_results = lsrro.solve(m, raise_on_failure=True) "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Optimization Results:\n",
+      "\n",
+      "----system metrics----\n",
+      "Feed: 1.05 kg/s, 66946 ppm\n",
+      "Product: 0.498 kg/s, 1000 ppm\n",
+      "Brine: 0.548 kg/s, 126890 ppm\n",
+      "Volumetric water recovery: 50.0%\n",
+      "Number of Stages: 3\n",
+      "Total Membrane Area: 265.86\n",
+      "Energy Consumption: 7.6 kWh/m3\n",
+      "Levelized cost of water: 1.71 $/m3\n",
+      "Primary Pump Capital Cost ($/m3):0.5047265838517887\n",
+      "Booster Pump Capital Cost ($/m3): 0.1318504520423674\n",
+      "ERD Capital Cost ($/m3):0.04116999386381932\n",
+      "Membrane Capital Cost ($/m3): 0.14321735175018754\n",
+      "Indirect Capital Cost ($/m3): 0.41048219075408143\n",
+      "Electricity cost ($/m3): 0.5323462487411291\n",
+      "\n",
+      "\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(\"Optimization Results:\\n\")\n",
+    "lsrro.display_system(m)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "lcow_breakdown_results, case_list = lsrro.get_lcow_breakdown(m, new_case=\"Optimization Results\", existing_case_list=case_list, results=lcow_breakdown_results)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 6: Visualize results"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Compare the breakdown of levelized cost of water for simulated and optimized results.\n"
```

Comment:
I think this table looks great! I'm assuming you'll verbally explain what goes into "Other OPEX"?

Reply:
Yep!